spdx-utils/src/main/kotlin/SpdxUtils.kt

```diff
@@ -32,12 +32,19 @@ import org.apache.commons.codec.digest.DigestUtils
  */
 val LICENSE_FILE_NAMES = listOf(
     "LICENSE*",
+    "license*",
```

Comment:
For me, the commit message lacks a rationale *why* we'd want to support lower-case license file names. By convention, these files should indeed be upper-case.

Also, instead of hard-coding the lower-case variants here, it probably makes more sense to programmatically generate them, e.g. as part of initializing `LICENSE_FILE_MATCHERS`.

Reply:
I have added a rationale to the commit mesage. I prefer to keep the plain list for simplicity, it's not a long list. Also, only using lowercase for the matchers could cause inconsistencies.

Reply:
Commit message typo in "convetion".

I've also seen projects that use a file named "License". Shouldn't we then also support that, and do a fully case-insensitive (glob) match on all platforms? Though I'm not yet sure how to elegantly implement that.

Reply:
Unfortunately globs do not support a case insensitive mode. I think it would be best to just add "License" and similar to the list.

Reply:
Now that we basically agreed that we need three variants, upper-case, lower-case and capitalized, I'd like to again propose a programmatic approach, this time directly when initializing `LICENSE_FILE_NAMES`:

```
val LICENSE_FILE_NAMES = listOf(
    "license*",
    "licence*",
    "unlicense",
    "unlicence",
    "copying*",
    "copyright",
    "patents"
).flatMap { listOf(it, it.toUpperCase(), it.capitalize()) }
```

Note that the list needs to contain the lower-case strings now, or else `capitalize()` would not work.

Reply:
Now that we basically agreed that we need three variants, upper-case, lower-case and capitalized, I'd like to again propose a programmatic approach, this time directly when initializing `LICENSE_FILE_NAMES`:

```
val LICENSE_FILE_NAMES = listOf(
    "license*",
    "licence*",
    "unlicense",
    "unlicence",
    "copying*",
    "copyright",
    "patents"
).flatMap { listOf(it, it.toUpperCase(), it.capitalize()) }
```

Note that the list needs to contain the lower-case strings now, or else `capitalize()` would not work.

Reply:
Done.